samples/cpp/benchmark_app/main.cpp

```diff
@@ -738,9 +773,16 @@ int main(int argc, char* argv[]) {
             // --------------------------------------------------------
             next_step();
             startTime = Time::now();
+            auto compileModelMemStart = getPeakMemoryUsage();
             compiledModel = core.compile_model(model, device_name, device_config);
+            auto compileModelMemEnd = getPeakMemoryUsage();
             duration_ms = get_duration_ms_till_now(startTime);
             slog::info << "Compile model took " << double_to_string(duration_ms) << " ms" << slog::endl;
+
+            slog::info << "Start of compilation memory usage: Peak " << compileModelMemStart << " KB" << slog::endl;
+            slog::info << "End of compilation memory usage: Peak " << compileModelMemEnd << " KB" << slog::endl;
+            slog::info << "Load model ram used " << compileModelMemEnd - compileModelMemStart << " KB" << slog::endl;
```

Comment:
Similar story as for [NPU compiler](https://github.com/openvinotoolkit/npu_compiler/blob/develop/src/vpux_compiler/src/compiler.cpp#L701)